src/components/LoginForm.vue

```diff
@@ -72,6 +72,7 @@ export default defineComponent({
     const router = useRouter()
     const store = useStore()
     const { t } = useI18n()
+    const className = 'login-form'
```

Comment:
Define `classes` outside Vue component. See example: https://github.com/Adamant-im/adamant-im/blob/67cb2a6e881f82e6330e4af321784f292a966b8b/src/components/nodes/NodesTable.vue#L96-L101

Reply:
Done in [4dd0a94](https://github.com/Adamant-im/adamant-im/pull/679/commits/4dd0a94ce58e78200a7e9f5c4fad44bb2ea661f0)